<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateAssetInfoDialog</class>
 <widget class="QDialog" name="UpdateAssetInfoDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>518</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Update Asset Information</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEditAssetName">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="3">
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="3">
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBoxRequiredExpense">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QRadioButton" name="radioButtonLiquid">
            <property name="text">
             <string>Liquid</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonIlliquid">
            <property name="text">
             <string>Illiquid</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelAssets">
       <property name="text">
        <string>Select Asset: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="comboBoxAssets">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <property name="minimumContentsLength">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Asset Name:    </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Liquidity:    </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_18">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_17">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="6">
      <widget class="QLineEdit" name="lineEditMonth8">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QLineEdit" name="lineEditMonth11">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="labelMonth5">
       <property name="text">
        <string>Month 1:  $</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="labelMonth7">
       <property name="text">
        <string>Month 1:  $</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QLabel" name="labelMonth12">
       <property name="text">
        <string>Month 1:  $</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLineEdit" name="lineEditMonth6">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLineEdit" name="lineEditMonth5">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLineEdit" name="lineEditMonth9">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="labelMonth9">
       <property name="text">
        <string>Month 1:  $</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEditMonth1">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEditMonth3">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QLineEdit" name="lineEditMonth10">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_16">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEditMonth4">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="labelMonth4">
       <property name="text">
        <string>Month 1:  $</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="labelMonth6">
       <property name="text">
        <string>Month 1:  $</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="labelMonth8">
       <property name="text">
        <string>Month 1:  $</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="6">
      <widget class="QLineEdit" name="lineEditMonth12">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="labelMonth2">
       <property name="text">
        <string>Month 1:  $</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QLabel" name="labelMonth11">
       <property name="text">
        <string>Month 1:  $</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelMonth1">
       <property name="text">
        <string>Month 1:  $</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLineEdit" name="lineEditMonth7">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="labelMonth3">
       <property name="text">
        <string>Month 1:  $</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLabel" name="labelMonth10">
       <property name="text">
        <string>Month 1:  $</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEditMonth2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonUpdateAsset">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDeleteAsset">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
